app/people/customer.py

```diff
@@ -0,0 +1,7 @@
+class Customer:
+    def __init__(self, name: str, food: str) -> None:
+        self.name = name
+        self.food = food
+
+    def watch_movie(self, movie: str) -> str:

```

Comment:
Invalid type hint for return type
```suggestion
    def watch_movie(self, movie: str) -> None:
```

Reply:
Done